main.nf

```diff
@@ -9,164 +9,19 @@
-                                    "strictly_unique_peptides" = use peptides mapping to a unique single protein only
-                                    "shared_peptides" = use shared peptides only for its best group (by inference score)
-
-    Statistical post-processing (currently only for feature-based quant.):
-      --skip_post_msstats           Skip MSstats for statistical post-processing?
-      --ref_condition               Instead of all pairwise contrasts, uses the given condition number (corresponding to your experimental design) as a reference and
-                                    creates pairwise contrasts against it (TODO fully implement)
-      --contrasts                   Specify a set of contrasts in a semicolon seperated list of R-compatible contrasts with the
-                                    condition numbers as variables (e.g. "1-2;1-3;2-3"). Overwrites "--reference" (TODO fully implement)
-      --add_triqler_output          Also create an output in Triqler's format for an alternative manual post-processing with that tool
-
-    Quality control:
-      --ptxqc_report_layout         Specify a yaml file for the report layout (see PTXQC documentation) (TODO fully implement)
-
-    Other options:
-      --outdir [file]                 The output directory where the results will be saved
-      --publish_dir_mode [str]        Mode for publishing results in the output directory. Available: symlink, rellink, link, copy, copyNoFollow, move (Default: copy)
-      --email [email]                 Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      --email_on_fail [email]         Same as --email, except only send mail if the workflow is not successful
-      --max_multiqc_email_size [str]  Threshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
-      -name [str]                     Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic
-
-    AWSBatch options:
-      --awsqueue [str]                The AWSBatch JobQueue that needs to be set when running on AWSBatch
-      --awsregion [str]               The AWS Region for your AWS Batch job to run on
-      --awscli [str]                  Path to the AWS CLI tool
-    """.stripIndent()
-}
 
-// Show help message
+////////////////////////////////////////////////////
+/* --               PRINT HELP                 -- */
+////////////////////////////////////////////////////
+
+def json_schema = "$projectDir/nextflow_schema.json"
 if (params.help) {
-    helpMessage()
+    def command = "nextflow run nf-core/proteomicslfq \
+  -profile <docker/singularity/conda/institute> \
```

Comment:
Missing podman (and charliecloud)

Reply:
Right, I copied from master. Oha charliecloud. What is this?

Reply:
The new kid on the block which is used at the Helmholtz Institutes in Munich. I don't know what advantages it has to be honest. https://github.com/hpc/charliecloud

There's an issue over at tools to add a config for it since Nextflow supports it

Reply:
https://nextflow.io/docs/edge/charliecloud.html
I seeeee

Reply:
I just added it. Looks better to have more haha